aquadoggo/src/graphql/client/utils.rs

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use async_graphql::{Pos, ServerError, ServerResult};
+use p2panda_rs::document::DocumentViewId;
+use p2panda_rs::schema::SchemaId;
+
+use crate::db::provider::SqlStorage;
+
+/// Validate that the given view matches the given schema.
+///
+/// Panics if the view id does not exist in the store.
+pub async fn validate_view_matches_schema(
```

Comment:
Should/will this validation be happening at a different point in our stack later on?

Reply:
Yes please!